uui/components/pickers/DataPickerFooter.tsx

```diff
@@ -52,6 +62,18 @@ function DataPickerFooterImpl<TItem, TId>(props: PropsWithChildren<DataPickerFoo
                     />
                 </FlexCell>
             )}
+
+            {(isSinglePicker && !view.selectAll) && (
```

Comment:
we can  remove !view.selectAll from this condition, because for signle select it's always will be false and moreover we have here only 'clear' button, not 'select all'